### ISSUE TITLE
Audio Stop Playing on RecyclerView Scroll

### ISSUE THREAD

**Issue by ravinderparihar** [rygelouv/android-audio-sensei#1]
Hi,
I am using AudioSenseiPlayerView in RecyclerView for multi audios but when i am scrolling RecyclerView, it stop playing audio, please describe how can i keep playing clicked audio on RecyclerView Scroll?

**Comment by rygelouv** [comment]
Hi @ravinderparihar i did not noticed that. Thank you for your feedback. Let me think dig into it a litter deeper and fix that as soon as possible. Stay tuned. Thank you again

**Comment by ravinderparihar** [comment]
Hi Rygel Louv,
Thanks to notice, have you got any solution regarding above issue?

**Comment by rygelouv** [comment]
Yes, working on apply a fix. Will publish a new version later on today.

**Comment by ravinderparihar** [comment]
Rygel Louv,

Thank you so much for your efforts and communication, please update when done.

**Comment by rygelouv** [comment]
@ravinderparihar i just published a new version (0.1.1) that fixes the problem. Please check the readme file, try it out and let me know. 
Be careful about the dependency name though, it moved from 'com.github.Rygelouv:android-audio-sensei:v0.1.0-beta' to compile 'com.github.rygelouv:android-audio-sensei:0.1.1'. 

Waiting for your feedback.

**Comment by ravinderparihar** [comment]
Rygel Louv,
On thing i want to suggest that, can you show progress bar while it fetching audio from server and the progress values of song below the seek bar, so that it will be more friendly with users, it is an suggestion if you can, thank you.

**Comment by ravinderparihar** [comment]
ok thanks

**Comment by ravinderparihar** [comment]
app in background keep crashing, please do something as soon as you can, either i have to change the dependency or have to find other solution.

**Comment by rygelouv** [comment]
Could you give some more insight please ?

**Comment by bstillitano** [comment]
@rygelouv I think that @ravinderparihar is asking for more than this library is built for. The library in it's current state is amazing and you should be proud of the work you've done. The only feedback I'd have is perhaps improving the stock UI a tiny bit but even that is just being picky. 

You've made an awesome library that's saved me a tonne of time.

**Comment by rygelouv** [comment]
@bstillitano thank you very much. I really appreciated it. And you guys are right. I'm currently quite busy with work. But i'm preparing to get a V2.0 of this library. I need to review everything, rewrite some parts of the code add some additional features and just like you suggest, I will improve the stock UI.

**Comment by bstillitano** [comment]
Awesome mate! Let me know if you need any help. I’d love to get involved.

**Comment by rygelouv** [comment]
I sure will

**Comment by abdellatifab** [comment]
can i play audio from url

**Comment by rygelouv** [comment]
@abdellatifab working on it. Sorry I don't put much time in this library anymore but I'm for sure preparing for a new release.
